Uppsala/UppEt12.xml

```diff
@@ -771,9 +819,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <bindingDesc>
                             <binding contemporary="true" xml:id="binding">
                                 <decoNote xml:id="b1" type="Boards">Wooden boards covered with
-                                    decorated leather.</decoNote>
+                                    blind-tooled leather.</decoNote>
                                 <decoNote xml:id="b2" type="EndLeaves" pastedown="L R UTI">Blue
-                                    patterned textile pastedowns.</decoNote>
```

Comment:
I think that type="EndLeaves" pastedown="L R UTI" does not apply to inlays. There should be a keyword textileInlay that can be used in the content of a decoNote for binding

Reply:
There should be one, but there isn't, correct?

Reply:
`<term key="textileInlay"/>` exists